mesonbuild/compilers/c.py

```diff
@@ -772,6 +772,10 @@ def find_library(self, libname, env, extra_dirs, libtype='default'):
             args = ['-l' + libname]
             if self.links(code, env, extra_args=args):
                 return args
+        if is_dragonflybsd():
+            args = ['-L/usr/local/lib', '-l' + libname]
```

Comment:
Do we maybe want to always search first for libraries in `/usr/local/lib` on dragonflybsd? The only case I can think of is when the same library is in both, and don't we want to prefer the ports version in that case?